Ficek_QuantiApp/Ficek_QuantiApp/Features/Sources/RocketDetail/RocketDetailCore.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+import ComposableArchitecture
+import Dependencies
+import RocketClient
+import RocketLaunch
+
+public struct RocketDetailCore: ReducerProtocol{
+    
+    public init() { }
+    
+    public struct State: Equatable, Identifiable {
+        public var id: String { rocket.id }
```

Comment:
Tady to mas nejak divne zformatovany... Ale formatting prijde v druhy vlne co? Kdyztak si to resolvni...